docs/faq/proof-of-stake/pos-general-faq.md

```diff
@@ -0,0 +1,119 @@
+# **<i class="fa fa-ticket"></i> PoS General FAQ**
+
+---
+
+#### **1. What is Proof-of-Stake mining?**
```

Comment:
I probably wouldn't talk about "PoS mining" anywhere.  Mining refers to searching for a computationally hard answer through brute force, while our stake system doesn't have that.  Perhaps:

>####  1. How does Decred's Proof-of-Stake system work?